django/db/models/expressions.py

```diff
@@ -620,6 +624,8 @@ def as_sql(self, compiler, connection):
 
 
 class RawSQL(Expression):
+    contributes_to_group_by = True
```

Comment:
I suggested it. Basically the raw sql could be just alias.col for some column of the query.